api/src/LinkyLink/Controllers/HealthController.cs

```diff
@@ -11,6 +11,7 @@ namespace LinkyLink.Controllers
     [ApiController]
```

Comment:
[SUG] For consistency, I would add summary annotations at the top of the HealthController and the Ping Method.

Reply:
Good catch.